client/my-sites/domains/domain-management/dns/dns-records.jsx

```diff
@@ -105,13 +112,21 @@ class DnsRecords extends Component {
 		);
 	}
 
+	renderPlaceholder() {
+		return config.isEnabled( 'domains/dns-records-redesign' ) ? (
+			<DomainMainPlaceholder breadcrumbs={ this.renderBreadcrumbs } />
+		) : (
+			<DomainMainPlaceholder goBack={ this.goBack } />
+		);
+	}
```

Comment:
I believe this component will only be rendered when the feature flag is enabled, as the feature flag check is at the controller level (`domain-management/controller.jsx`). So I think you can just return `<DomainMainPlaceholder breadcrumbs={ this.renderBreadcrumbs } />` here.

Reply:
Thanks for that. 
Confirmed that the changes didn't affect the page without the feature flag: 

![image](https://user-images.githubusercontent.com/18705930/142252304-f166e53a-1c73-4e4f-a170-b2d7d292bcbc.png)